PowerEditor/src/Utf8_16.cpp

```diff
@@ -171,7 +171,10 @@ size_t Utf8_16_Read::convert(char* buf, size_t len)
 					delete [] m_pNewBuf;
                 m_pNewBuf  = NULL;
                 m_pNewBuf  = new ubyte[newSize];
-				m_nAllocatedBufSize = newSize;

```

Comment:
File has mixed indentation used tabs for nullpointer checks.
@donho Indentation should be changed to one style within this file.